src/xcode/ENA/ENA/Source/Scenes/ExposureDetection/Survey/SurveyURLProvider.swift

```diff
@@ -51,6 +57,10 @@ enum SurveyError: Error {
 			return String(format: AppStrings.SurveyConsent.errorChangeDeviceTime, errorCode)
 		case .alreadyParticipated(let errorCode):
 			return String(format: AppStrings.SurveyConsent.errorAlreadyParticipated, errorCode)
+		case .submissionTooEarly, .minimumTimeSinceOnboarding:
+			//these cases are un related to the survay
```

Comment:
// these cases are unrelated to the survey

Reply:
```suggestion
			/ /these cases are unrelated to the survey
```